Jenkinsfile

```diff
@@ -356,7 +356,8 @@ timestamps { notifyStatusChange(
       if (env.BRANCH_NAME == primaryBranch) {
         echo "Deploying Storybook"
         tinyGit.withGitHubSSHCredentials {
-          exec('yarn -s --cwd modules/oxide-components deploy-storybook')
+           echo "Deploying Storybook skipped until the Jenkinsfile is fixed"
```

Comment:
Is there a Jira we can mention or at least as a comment?

Reply:
This is very much temporary, we intend to fix this within this week. The Jira ticket is still in development. So this is not a tech debt. Does that make sense ?